libs/common-services/src/main/java/uk/gov/di/ipv/core/library/exceptions/GetIpvSessionException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.di.ipv.core.library.exceptions;
+
+public class GetIpvSessionException extends RuntimeException {
```

Comment:
I don't think these should be `RuntimeException`s - we want to catch them!

Reply:
Was catching it where needed in handler where retry is needed. 
Yeah was avoiding changes in other places where retry is not used for get Ipv session. 
Done.